packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js

```diff
@@ -75,126 +92,128 @@ const ApplicationInfosPage = () => {
     <Layout>
       <SettingsPageTitle name="Application" />
       <Main>
-        <HeaderLayout
-          title={formatMessage({ id: 'Settings.application.title', defaultMessage: 'Overview' })}
-          subtitle={formatMessage({
-            id: 'Settings.application.description',
-            defaultMessage: 'Administration panel’s global information',
-          })}
-          primaryAction={
-            <Button onClick={handleSubmit} startIcon={<Check />}>
-              {formatMessage({ id: 'global.save', defaultMessage: 'Save' })}
-            </Button>
-          }
-        />
-        <ContentLayout>
-          <Stack spacing={6}>
-            <Box
-              hasRadius
-              background="neutral0"
-              shadow="tableShadow"
-              paddingTop={6}
-              paddingBottom={6}
-              paddingRight={7}
-              paddingLeft={7}
-            >
-              <Stack spacing={5}>
-                <Typography variant="delta" as="h3">
-                  {formatMessage({
-                    id: 'global.details',
-                    defaultMessage: 'Details',
-                  })}
-                </Typography>
+        <form onSubmit={handleSubmit}>
+          <HeaderLayout
+            title={formatMessage({ id: 'Settings.application.title', defaultMessage: 'Overview' })}
+            subtitle={formatMessage({
+              id: 'Settings.application.description',
+              defaultMessage: 'Administration panel’s global information',
+            })}
+            primaryAction={
+              canRead && canUpdate ? (
+                <Button type="submit" startIcon={<Check />}>
+                  {formatMessage({ id: 'global.save', defaultMessage: 'Save' })}
+                </Button>
+              ) : null
```

Comment:
```suggestion
              canRead && canUpdate && (
                <Button type="submit" startIcon={<Check />}>
                  {formatMessage({ id: 'global.save', defaultMessage: 'Save' })}
                </Button>
              )
```

This means without these permissions the form does not have a submit button, but using the keyboard it can still be submitted. Do you also have to remove the onSubmit handler on the form?

Reply:
I cannot reproduce submit using the keyboard 🤔 anyway it makes sense and it's a good catch, my bad! 🙏